pkg/clustertree/knode-manager/adapters/interface.go

```diff
@@ -20,6 +20,12 @@ type PodHandler interface {
 	List(context.Context) ([]*corev1.Pod, error)
 
 	Notify(context.Context, func(*corev1.Pod))
+
+	UpdateClientSecret(old, new *corev1.Secret) (*corev1.Secret, error)
+
+	DeleteClientSecret(namespace string, secretName string) error
+
+	CreateClientSecret(namespace string, secretName string) error
```

Comment:
why this three function in PodHandler interface?

Reply:
https://github.com/kosmos-io/kosmos/pull/84#discussion_r1349607849.
Brothoer Zhou Hao suggested I put it in the podhandler.
And I think when the pod is created, the secret is also created. So I also think it can stay in PodHandler

Reply:
I think it's not appropriate

Reply:
I re-created SecretHandler